code/game/objects/structures/reflector.dm

```diff
@@ -0,0 +1,71 @@
+//dir 1 = north
+//dir 2 = south
+//dir 8 = west
+//dir 4 = east
```

Comment:
Because previously it used magic numbers for dirs. Comments from then haven't been removed, evidently.